Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM python:3.11.11-slim
+
+ENV PYTHONBUFFERED 1
+
+WORKDIR app/
```

Comment:
The `WORKDIR` should be an absolute path. Consider changing it to `/app` instead of `app/`.